pkg/queue/manager.go

```diff
@@ -329,12 +329,14 @@ func (m *Manager) AddOrUpdateWorkloadWithoutLock(w *kueue.Workload) bool {
 	qKey := workload.QueueKey(w)
 	q := m.localQueues[qKey]
 	if q == nil {
+		fmt.Println("LocalQueue for workload not found")
 		return false
 	}
 	wInfo := workload.NewInfo(w, m.workloadInfoOptions...)
 	q.AddOrUpdate(wInfo)
 	cq := m.hm.ClusterQueues[q.ClusterQueue]
 	if cq == nil {
+		fmt.Println("ClusterQueue for workload not found")
```

Comment:
```suggestion
		return ErrClusterQueueDoesNotExist
```

Reply:
if we do this then we have to change the type of the function cause it was returning a bool and after adding this it should return error .. 
so should I add it like you mentioned or anything other?

Reply:
Yes, sure.

Reply:
okay!